addon/templates/components/lt-body.hbs

```diff
@@ -12,7 +13,7 @@
         {{yield columns rows}}
       {{else}}
         {{! scrollbar.update action when rows changes and evaluates to rows !}}
```

Comment:
Yep, I should remove. I'm not sure if change is 100% safe, but if it's not I'll address it.